.github/workflows/operator-integration-test.yml

```diff
@@ -55,27 +55,14 @@ jobs:
           make deploy
           kubectl wait --for=condition=Ready pod --all -n amazon-cloudwatch
 
-      - name: Test case for cloudwatch agent pod creation
```

Comment:
why was this removed?

Reply:
This test felt repetitve because the other part of the integ tests pod creation, but I can see why this might be useful and will add it back.